### PR TITLE
Update README.md to include EclipseLink 5 information

### PR DESCRIPTION
**Pull request by velo**

### CODE REVIEW

**Comment by rfelcman** [comment]
Please accept https://api.eclipse.org/git/eca/status/gh/eclipse-ee4j/eclipselink/2206

**Comment by velo** [comment]
> Please accept https://api.eclipse.org/git/eca/status/gh/eclipse-ee4j/eclipselink/2206

done

File: README.md [hunk 1]
```diff
@@ -11,11 +11,16 @@
...
 
-EclipseLink master (4.0.x)
-[![Maven Central](https://img.shields.io/maven-central/v/org.eclipse.persistence/eclipselink.svg?versionPrefix=4.0&label=Maven%20Central)](https://mvnrepository.com/artifact/org.eclipse.persistence/eclipselink)
+EclipseLink master (5.0.x)
+[![Maven Central](https://img.shields.io/maven-central/v/org.eclipse.persistence/eclipselink.svg?versionPrefix=5.0&label=Maven%20Central)](https://mvnrepository.com/artifact/org.eclipse.persistence/eclipselink)
 [![Jakarta Staging (Snapshots)](https://img.shields.io/nexus/s/https/jakarta.oss.sonatype.org/org.eclipse.persistence/eclipselink.svg)](https://jakarta.oss.sonatype.org/content/repositories/staging/org/eclipse/persistence/eclipselink)
 [![Master Build Status](https://ci.eclipse.org/eclipselink/job/eclipselink-nightly-master/badge/icon)](https://ci.eclipse.org/eclipselink/job/eclipselink-nightly-master)
 
+
+EclipseLink (4.0.x)
```
Comment (rfelcman) [inline README.md:20]: Please remove braces there.

File: README.md [hunk 2]
```diff
@@ -11,11 +11,16 @@
 [![Language grade: Java](https://img.shields.io/lgtm/grade/java/g/eclipse-ee4j/eclipselink.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/eclipse-ee4j/eclipselink/context:java)
 [![Total alerts](https://img.shields.io/lgtm/alerts/g/eclipse-ee4j/eclipselink.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/eclipse-ee4j/eclipselink/alerts/)
 
-EclipseLink master (4.0.x)
-[![Maven Central](https://img.shields.io/maven-central/v/org.eclipse.persistence/eclipselink.svg?versionPrefix=4.0&label=Maven%20Central)](https://mvnrepository.com/artifact/org.eclipse.persistence/eclipselink)
+EclipseLink master (5.0.x)
```
Comment (rfelcman) [inline README.md:14]: Please remove braces there to be consistent with 3.0, 2.7

**Review by rfelcman — COMMENTED** [review]
And update copyright year 2022 -> 2024 at line 1

**Review by rfelcman — APPROVED** [review]
LGTM

**Pull request merged**